cmd/es-rollover/app/actions.go

```diff
@@ -35,30 +37,54 @@ type Action interface {
 	Do() error
 }
 
+type ClientConfig struct {
+	configtls.ClientConfig `mapstructure:",squash"`
+	Enabled                bool
+}
+
+func (c *ClientConfig) AddFlags(flags *flag.FlagSet) {
```

Comment:
this is redundant, we should continue using logic from tlscfg for this (see how it's done in cmd/query)